### ISSUE TITLE
Add "token-encrypted" : true/false in configuration and change DB Schema accordingly

### ISSUE THREAD

**Issue by dipak-pawar** [fabric8-services/fabric8-cluster#38]
Currently, we get encrypted tokens in cluster service configuration (`oso-cluster.conf`, an OenShift secret mounted as volume), but since the cluster configuration is stored in a DB which is also encrypted, there is no need to encrypt the token retrieve during the creation of other clusters using the `POST /api/clusters/` API.

In order to track the data encryption state, we need to add an extra param in the `cluster` table and in the configuration file as well: `token-encrypted` (`true`/`false`). Also, since we don't have any `sa-token-encypted` value in our configuration (`oso-cluster-configuration.json`), let's set it to `true` as all tokens in configuration are encrypted and we don't want to update secret again .

So in short set true if not present, otherwise set it to it's value
